src/Constructorio_NET.Tests/client/modules/CatalogTests.cs

```diff
@@ -216,6 +216,101 @@ public async Task PatchCatalogWithItemGroups()
         }
 
         [Test]
+        public async Task RetrieveSearchabilitiesShouldReturnResult()
+        {
+            ConstructorIO constructorio = new ConstructorIO(this.Config);
+            RetrieveSearchabilitiesRequest req = new RetrieveSearchabilitiesRequest();
+            SearchabilitiesResponse res = await constructorio.Catalog.RetrieveSearchabilities(req);
+            Assert.IsNotNull(res.TotalCount, "Total Count should exist");
+            Assert.IsNotNull(res.Searchabilities, "Searchabilities should exist");
+        }
+
+        [Test]
+        public async Task RetrieveSearchabilitiesWithOptionalParamsShouldReturnResult()
+        {
+            ConstructorIO constructorio = new ConstructorIO(this.Config);
+            RetrieveSearchabilitiesRequest req = new RetrieveSearchabilitiesRequest
+            {
+                Filters = new Dictionary<string, string> { { "name", "groups" } },
+                Page = 1,
+                NumResultsPerPage = 1,
+                SortOrder = "descending",
+                SortBy = "name",
+                Searchable = true
+            };
+            SearchabilitiesResponse res = await constructorio.Catalog.RetrieveSearchabilities(req);
+            Assert.IsTrue(res.TotalCount == 1, "Total Count should exist");
+            Assert.IsNotNull(res.Searchabilities.Count == 1, "Searchabilities should exist");
```

Comment:
The assertion might need to be changed to `Assert.IsTrue` since we're testing an expression here. The description might need an update as well. Same thing for the the tests below that's using the same pattern.